openslides_backend/shared/otel.py

```diff
@@ -0,0 +1,62 @@
+import os
+from contextlib import nullcontext
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+from opentelemetry.instrumentation.requests import RequestsInstrumentor
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from typing import Any, Dict
+
+from .env import is_truthy
+
+
+def init(service_name: str) -> None:
+    """
+    Initializes the opentelemetry components and connection to the otel collector.
+    """
+    if not is_truthy(os.environ.get("OPENTELEMETRY_ENABLED", "false")):
```

Comment:
Define `OPENTELEMETRY_ENABLED` as its own variable in `env.py` (or maybe diectly use a explicit helper function, similar to `is_dev_mode`

Reply:
Use two helper function in env.py to get otel url and is otel enabled.